### DISCUSSION TITLE
Adding values

### DISCUSSION THREAD

**Discussion by fvirgola80** [poblabs/weewx-belchertown#837] [Ideas]
The idea is to be able to add the values
wind direction, pressure, millimeters or inc for rain and uv in 24h 3h and 1h with commit #795 in addition to those already added for dew poit and humidity
@ThePeePs

**Comment by ThePeePs** [comment]
Any fields that are in the forecast data can be used and displayed, you would just need to edit the [javascript](https://github.com/poblabs/weewx-belchertown/blob/master/skins/Belchertown/js/belchertown.js.tmpl#L1197) to have it show that data you want instead of the dewpoint or humidity.  I do know some people that have changed the POP to be the amount of rain (either inches or mm) instead.

**Comment by fvirgola80** [comment]
thank you very much for the answer, yes I have seen some users who changed the variables obtaining a different result. I leave you my email address fvirgola80@gmail.com I would like to give you some of my ideas.

**Comment by James-76** [comment]
This did come up in #793, and I used the great bit of code from @norecords, with a little tweak, to include precip, so I'm very much for this.
